frontend/src/app/applications/permit-application-list/permit-application-list.component.html

```diff
@@ -47,7 +47,8 @@ <h3 class="usa-alert-heading">Warning</h3>
           <span *ngIf="application.status === 'Review'" class="usa-label">{{reviewText}}</span>
         </div>
         <label *ngIf="isAdmin">Type</label>
-        <div class="application-type" *ngIf="isAdmin">{{ application.type }}</div>
+        <div class="application-type" *ngIf="isAdmin && application.type !== 'Temp outfitter'">{{ application.type }}</div>
+        <div class="application-type" *ngIf="isAdmin && application.type === 'Temp outfitter'">Temporary Outfitting & Guiding</div>
```

Comment:
Looking good! Your code should do the job just fine, but if you're interested you could try consolidating it into this:

```javascript
<div class="application-type" *ngIf="isAdmin">
    {{application.type === 'Temp outfitter' ? 'Temporary Outfitting & Guiding' : application.type}}
</div>
```

Reply:
Aaah, ternary operator. I didn't think about that there. Thank you!